betty/extension/webpack/jinja2/filter.py

```diff
@@ -0,0 +1,25 @@
+"""
+Provide Jinja2 filters to integrate with Webpack.
+"""
+from __future__ import annotations
+
+from jinja2 import pass_context
+from jinja2.runtime import Context
+
+from betty.jinja2.filter import filter_public_js
+
+
+@pass_context
+def filter_webpack_js_entrypoint(context: Context, entrypoint: str) -> None:
+    """
+    Add a JavaScript file to the current page.
+    """
+    filter_public_js(context, '/js/main.js')
+    # @todo
+    # @todo Add the entrypoint to a data attribute or something on the page, so main.js can work with it
```

Comment:
Finish this. Can we collect all entrypoint registrations and add them to the body?